### PR TITLE
AQCU-1200: change displayed qualifier value to 'code - display name' …

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
…for TSS.

### CODE REVIEW

File: tests/testthat/test-timeseriessummary.R [hunk 1]
```diff
@@ -92,6 +92,7 @@ test_that('parseTSSRealtedSeries properly retrieves the related upchain series',
 
 test_that('parseTSSQualifiers properly retrieves the qualifiers', {
   timezone <- "Etc/GMT+5"
+  browser()
```
Comment (zmoore-usgs) [inline tests/testthat/test-timeseriessummary.R]: Should probably remove this guy

**Review by zmoore-usgs — APPROVED** [review]
Changes look good to me. Pulled and tested locally and it works as expected!

**Comment by zmoore-usgs** [comment]
Guessing that drop might be caused by that browser call messing up tests following it?

File: tests/testthat/test-timeseriessummary.R (hunk 1 above)
Comment (kmschoep-usgs) [reply tests/testthat/test-timeseriessummary.R]: whoops, yes.

**Pull request merged**